protocols/gossipsub/src/metrics.rs

```diff
@@ -179,10 +180,10 @@ pub(crate) struct Metrics {
     /// topic. A very high metric might indicate an underperforming network.
     topic_iwant_msgs: Family<TopicHash, Counter>,
 
-    /// The size of the priority queue.
-    priority_queue_size: Histogram,
+    /// The size of the priority queue for each PeerId.
+    priority_queue_size: Family<PeerIdLabel, Histogram, HistBuilder>,
```

Comment:
If we do this, doesn't this grow unbounded. Like PeerId's come and go. And we have no way to remove them from the mappings?

Reply:
This was for debugging purposes more than anything. Can be removed now i think